dsl/src/main/java/tech/pegasys/peeps/node/Account.java

```diff
@@ -63,4 +65,12 @@ public static Map<GenesisAddress, GenesisAccount> of(final Account... accounts)
 
     return Collections.unmodifiableMap(mapped);
   }
+
+  public Credentials credentials() {
```

Comment:
Why does the credentials class need to be exposed? We already have the address and methods for signing.

Reply:
Because I wanted to use web3j :smile: and sign the transactions locally and use send raw directly.